docs/yellowpaper/sections/03-sphinx-packet-format/index.tex

```diff
@@ -89,14 +89,11 @@ \subsubsection{Routing information}
 
 $\beta_i$ is computed as the concatenation of $Z$ and a sequence of padding which is then encrypted by XORing with the output of a PRG seeded with shared key $\varphi_{v-1}$ of node $v-1$. The result is finally concatenated with $\phi$ to ensure the header packets remain constant in size.
 
-In the original Sphinx paper, $Z$ is concatenated with an identifier $I$
-and $0$ padding sequence, where $I$ is used for SURBs (single-use reply
-blocks) such that $I \in \{0, 1\}^\kappa$. We do not use $I$ since HOPR does not
-currently employ SURBs. We do, however, include $hint$ and $challenge$ values in
-$\beta$, defined in the \nameref{sec:proofofrelay} section. These values are not included in the original Sphinx paper but are needed for the HOPR protocol.
+In the original Sphinx paper, $Z$ is concatenated with an identifier $I$ and $0$ padding sequence, where $I$ is used for SURBs (single-use reply
+blocks) such that $I \in \{0, 1\}^\kappa$. We do not use $I$ since HOPR does not currently employ SURBs. We do, however, include $hint$ and $challenge$ values in
```

Comment:
The relevant information here is that we add PoR to the beta part and not that we drop capital I. For people who haven't read the SPHINX paper in detail, it might remain unclear why there is any capital I and at this point we take a lot of attention from the reader saying that we are not using it.